include/SDL2/SDL_stdinc.h

```diff
@@ -290,7 +290,7 @@ typedef uint64_t Uint64;
 #define SDL_PRIX64 PRIX64
 #elif defined(__WIN32__) || defined(__GDK__)
 #define SDL_PRIX64 "I64X"
-#elif defined(__LINUX__) && defined(__LP64__)
+#elif defined(__LP64__) && !defined(__APPLE__)
```

Comment:
SDL_stdinc.h changes should be applied to main SDL2 and SDL3 branches, no?

Reply:
Only SDL2, I copied this from SDL3.

Reply:
OK, let's do that in SDL2, so there won't be a confusing diff between SDL2 and sdl2-compat.

Reply:
Applied in SDL2: https://github.com/libsdl-org/SDL/commit/5abcfdb93e675d46341b4c2c74a1668afe7b98e1